docs/server/versions-dependencies.md

```diff
@@ -40,16 +40,18 @@ Temporal offers official support for, and is tested against, dependencies with t
 ### Persistence
 
 The only required Temporal Server dependency is a database, and there are multiple types of databases that are supported.
-Below we show specific versions used in our nightly test pipelines and actively tested before we release any version of Temporal.
+Temporal tests compatibility by spanning the **minimum** and **maximum** stable non-EOL major versions for each supported database. 
+As of time of writing, these specific versions used in our nightly test pipelines and actively tested before we release any version of Temporal:
```

Comment:
We don't need to specify "nightly" here: we have an assortment of pipelines that run on different triggers

Reply:
```suggestion
As of time of writing, these specific versions used in our test pipelines and actively tested before we release any version of Temporal:
```